AvaloniaApp/Services/SubmissionService.cs

```diff
@@ -0,0 +1,138 @@
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json;
+using Apachi.AvaloniaApp.Data;
+using Apachi.Shared.Crypto;
+using Apachi.Shared.Dtos;
+using Apachi.ViewModels.Auth;
+using Apachi.ViewModels.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace Apachi.AvaloniaApp.Services;
+
+public class SubmissionService : ISubmissionService
+{
+    private readonly ISession _session;
+    private readonly Func<AppDbContext> _dbContextFactory;
+    private readonly IHttpClientFactory _httpClientFactory;
+
+    public SubmissionService(
+        ISession session,
+        Func<AppDbContext> dbContextFactory,
+        IHttpClientFactory httpClientFactory
```

Comment:
Did you want to remove this? I recall you mentioned it wasn't necessary for SubmissionService to have access to this, nor does it seem like you're using it in the code anyways

Reply:
It is being used in the code. The code uses the private fields.

Reply:
Up to you, I just remember you saying it was redundant

Reply:
No they are not and I didn't say that.